PowerPointLabs/PowerPointLabs/Settings.StyleCop

```diff
@@ -274,22 +274,22 @@
         </Rule>
         <Rule Name="SystemUsingDirectivesMustBePlacedBeforeOtherUsingDirectives">
```

Comment:
No experience with stylecop or C#. As an outsider, I feel it is better to see the enabled rules in the config file. That's what we do in checkstyle config files.